UTM/main.m

```diff
@@ -43,7 +44,16 @@ kern_return_t catch_exception_raise(mach_port_t exception_port,
     return NULL;
 }
 
+static BOOL am_i_being_debugged() {
```

Comment:
better to use csops to check for CS_DEBUGGED here, in case it was set by a jailbreak already (as jailbreaks can set the codesigning flag without having to call ptrace)

Reply:
I don’t see why this case is relevant, as I would expect a version designed for jailbroken devices to be signed with dynamic-codesigning rather than doing tricks with `CS_DEBUGGED`…

Reply:
2 things about the dynamic-codesigning entitlement

To actually use it, the app needs to use mmap with MAP_JIT, and it needs to be inside a sandbox.

Without MAP_JIT, you’re basically using ptrace/CS_DEBUGGED

Reply:
I’m aware of those requirements; I don’t see either of them being an issue in this case?

Reply:
On that note, I should probably send a fix for https://github.com/hrydgard/ppsspp/issues/9186 to the PPSSPP people as well.